flycheck.el

```diff
@@ -3187,6 +3197,13 @@ Return ERRORS, modified in-place."
           errors)
   errors)
 
+(defun flycheck-relevant-error-other-file-p (err)
+  "Determine whether ERR is a relevant error for another file."
+  (let ((file-name (flycheck-error-filename err)))
+    (and buffer-file-name file-name
```

Comment:
Do we really need to have a a buffer-file-name here?

Reply:
Yes, because `buffer-file-name` can be nil, in which case it's not safe to call the function which checks that the files are the same.

Reply:
Right, but we could also not call that function in that case (if I'm editing a scratch file and that file doesn't compile because of an error in an import, I might still be interested in that error)

Reply:
Ah, I see! I think you're right. Will fix that in a few hours - currently at work. :-)

Reply:
Thanks!